dplace_app/static/js/colorMapService.js

```diff
@@ -68,15 +68,15 @@ function ColorMapService() {
 
     //blue to red gradient for environmental variables
     this.tempColor = function(index, min, max, name) {
-    if (name == "Monthly Mean Net Primary Production" || name == "Mean Growing Season NPP") {
-        hue = 30 + (((index - min) / (max - min)) * 88);
-    } else if (name == "Annual Mean Precipitation"){
-        color = this.mapColorMonochrome(min, max, index, 252);
-        return color;
-    }
-    else {
-        hue = 240 - (((index - min) / (max - min))*240);
-    }
+        if (name == "Monthly Mean Net Primary Production" || name == "Mean Growing Season NPP") {
+            hue = 30 + (((index - min) / (max - min)) * 88);
+        } else if (name == "Annual Mean Precipitation"){
```

Comment:
I think it would be good if we could replace such code with some sort of config lookup, maybe an additional column `legend_config` in `variables.csv`?